doc/guides/test/sanitycheck.rst

```diff
@@ -390,6 +395,16 @@ harness_config: <harness configuration options>
              tags: sensors
              depends_on: i2c
 
+    The following is an example yaml file with pytest harness_config options.
+
+    ::
+
+        tests:
+          test_section:
+          harness: pytest
+          harness_config:
+            handle_script: tests/.../test_pytest.py
```

Comment:
what if there is more than 1 file? I think it makes more sense to point to a directory with tests (and have a default, say 'pytest') and then call pytest on this directory and run all the available tests.

Reply:
Yes,  thanks for your suggestion,  I have changed this to use pytest directory.